tools/doc/html.js

```diff
@@ -149,21 +149,39 @@ function parseText(lexed) {
 // just update the list item text in-place.
 // lists that come right after a heading are what we're after.
 function parseLists(input) {
-  var state = null;
-  var depth = 0;
-  var output = [];
+  let state = null;
+  let depth = 0;
+  let output = [];
+  
+  let headingIndex = -1;
+  let heading = null;
+
   output.links = input.links;
-  input.forEach(function(tok) {
+  
+  input.forEach(function(tok, index) {
     if (tok.type === 'code' && tok.text.match(/Stability:.*/g)) {
+      if(heading && (index - 1 === headingIndex || index - 2 === headingIndex)) {
```

Comment:
can you run `make -j4 test`? It runs the linter, which I think also lints these files.